pkg/core/localize/locales/en/cmd/context.en.toml

```diff
@@ -3,28 +3,33 @@ one='Group, share and manage your rhoas services'
 
 [context.cmd.longDescription]
 one='''
-rhoas context commands allow developers to:
+Group your application service instances into service contexts.
 
-  * Group services into contexts that can be used with a number of CLI commands.
-  * Manage different service contexts by switching, listing and removing service contexts 
-  * Share context with others to use the same set of services
-  * Generating configuration for connecting to the services from various platforms and tools
+A service context is a group of application service instances and their configuration details. By using service contexts, you can group together application service instances that you want to use together.
 
-By default context is kept under users configuration folder. Users can view context location by executing "rhoas context status" command.
+After creating a service context, you can share it with other developers so that they can use the same group of application service instances.
 
-Additionally developers can specify custom location for contexts by setting $RHOAS_CONTEXT environment variable in their profile.
-Setting $RHOAS_CONTEXT to "./rhoas.json" will load contexts from the current folder.
+You can also use the service context to automatically generate configuration files that you need to use those application service instances in other development platforms and tools. For example, the service context can generate the following types of configurations:
```

Comment:
Pro tip. For very long lines we can use different toml syntaxes.
Simplest way is to break like with `\`

Reply:
:+1: